scripts/lerna-version.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
```

Comment:
**Suggestion:** Consider adding a shebang at the top of the script to specify the interpreter for executing the script. This will ensure that the script is executed using the correct version of bash, even if the user has multiple versions installed or has a different shell set as their default.
```suggestion
#!/usr/bin/env bash
```